ttnn/cpp/ttnn/operations/eltwise/binary/device/binary_composite_op.cpp

```diff
@@ -392,4 +392,12 @@ Tensor _polyval(const Tensor& input_a, const std::vector<float>& coeffs, const s
     return final_tensor;
 }
 
+Tensor ExecuteBinaryGE::operator()(const Tensor& input_a, const Tensor& input_b) {
+    return ttnn::ge(input_a, input_b, std::nullopt, std::nullopt, input_a);
+}
+
+Tensor ExecuteBinaryGE::operator()(const Tensor& input, float scalar) {
+    return ttnn::ge(input, scalar, std::nullopt, std::nullopt, input);
+}
```

Comment:
mm, I am not clear on what is going on here. 
we already have ttnn::ge, now we add another.. Ge operation, and register it as ge_ (inplace?) and remove memory_config? whats the logic?

Reply:
why is it defined here, instead of in binary?

Reply:
@ayerofieiev-tt 
An in-place operation is an operation that changes directly the content of a given Tensor without making a copy. Inplace operations in pytorch are always postfixed with a _, like .le_() or ge_() [Refernce](https://pytorch.org/docs/stable/generated/torch.Tensor.le_.html). 
 
In our case, @tt-aho and @eyonland suggested implementing this functionality using the optional output tensor support present in the standard versions of these operations. Consequently, the output_mem_config can be removed as it is no longer necessary.

Reply:
> why is it defined here, instead of in binary?

Will move it to binary.

Reply:
Please create a new `struct InplaceRelationalBinary {`  in binary.hpp and binary.cpp that does not accept the memory_config or an optional_output_tensor argument.
I want to get rid of template argument `, bool in_place` entirely and use this new version on all in-place ops.